src/vaadin-grid-column.html

```diff
@@ -580,10 +677,12 @@
             value: false
           },
 
+          /** @private */
           _bodyTemplate: {
             type: Object
           },
 
+          /** @private */
           _cells: Array
```

Comment:
These two seem to be used in other files too so should probably be `@protected`?

Reply:
Done.